elyra/metadata/metadata_app.py

```diff
@@ -376,6 +378,76 @@ def start(self):
             print(f"No instances of schemaspace {self.schemaspace} were migrated.")
 
 
+class SchemaspaceExport(SchemaspaceBase):
+    """Handles the 'export' subcommand functionality for a specific schemaspace."""
+
+    schema_name_option = CliOption("--schema_name", name='schema_name',
+                                   description='The schema_name of the metadata instances to export',
+                                   required=False)
+
+    valid_only_flag = Flag("--valid-only", name='valid-only',
+                           description='Only export valid instances (default includes invalid instances)',
+                           default_value=False)
+
+    clean_flag = Flag("--clean", name='clean',
+                      description='Clear out contents of the export directory',
+                      default_value=False)
+
+    directory_option = CliOption("--directory", name='directory',
+                                 description='The local file system path where the exported metadata will be stored',
+                                 required=True)
+
+    # 'Export' flags
+    options: List[Option] = [schema_name_option, valid_only_flag, clean_flag, directory_option]
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.metadata_manager = MetadataManager(schemaspace=self.schemaspace)
+
+    def start(self):
+        super().start()  # process options
+
+        include_invalid = not self.valid_only_flag.value
+        schema_name = self.schema_name_option.value
+        directory = self.directory_option.value
+        clean = self.clean_flag.value
+
+        try:
+            if(self.schema_name_option is not None):
+                metadata_instances = self.metadata_manager.get_all(include_invalid=include_invalid,
+                                                                   of_schema=schema_name)
+            else:
+                metadata_instances = self.metadata_manager.get_all(include_invalid=include_invalid)
+        except MetadataNotFoundError:
+            metadata_instances = None
+
+        if not metadata_instances:
+            print("No metadata instances found for schemaspace {}".format(self.schemaspace) +
```

Comment:
[Code style] Although we haven't been consistent in the past, we have started to 
- surround identifiers with single quotes to make it easier to read messages
- use f-strings instead of '...'.format(...)

Could you please update your code to follow the (undocumented) convention? 
```
print("No metadata instances found for schemaspace {}".format(self.schemaspace) +
```
 
to 
```
print(f"No metadata instances found for schemaspace '{self.schemaspace}'" ...
```
```suggestion
            print("No metadata instances found for schemaspace {}".format(self.schemaspace) +
```

Reply:
will do 👍

Reply:
Updated print statements to use f-strings instead of format and surrounded identifiers with single quotes.